### PR TITLE
Add a new nested e2e-aws job on the hypershift-ci-1 cluster

### PR DESCRIPTION
**Pull request by ironcladlou**

### CODE REVIEW

**Comment by ironcladlou** [comment]
Panic observed in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/26220/rehearse-26220-pull-ci-openshift-hypershift-main-e2e-aws-hypershift-ci-1/1493973361829613568

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml [hunk 1]
```diff
@@ -599,6 +599,148 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-gcp
+- as: e2e-aws-hypershift-ci-1
```
Comment (sjenning) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml:602]: `e2e-aws-nested`?

**Comment by sjenning** [comment]
/lgtm
/hold
on e2e-aws-nested rehearse passing

**Comment by sjenning** [comment]
passed
/hold cancel

**Pull request merged**